TheMET/TheMET/AboutAppViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AboutAppViewCell.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 11/10/2023.
+//
+
+import Foundation
+import UIKit
+
+class AboutAppViewCell: UITableViewCell {
+    
+    var label: UILabel = UILabel()
```

Comment:
Views (Cells in particular) must not expose access to their subviews. Make this a `private let` property. Then create a `var titleText: String?` computable property OR a `set(titleText: String?)` method to apply title-text.